### PR TITLE
INFRASYS-6951: Updated requirements after Salt Upgrade to 2015.8.7

### PR DESCRIPTION
**Pull request by vmadura**
INFRASYS-6951: Updated requirements after Salt Upgrade to 2015.8.7

### CODE REVIEW

**Comment by taoistmath** [comment]
+1

**Comment by adamjkeller** [comment]
+1

File: requirements.txt [hunk 1]
```diff
@@ -4,9 +4,9 @@ django-nose==1.3
 logilab-common==0.63.2
 mock==1.0.1
 nose==1.3.4
-pylint==1.4.0
+pylint==1.4.1
 redis==2.10.3
-salt==0.17.1
+salt==2015.8.7
```
Comment (kantselovich) [inline requirements.txt:9]: it probably needs to be latest

Comment (adamjkeller) [inline requirements.txt:9]: I disagree, why not match the current version we're using? What if there are major functionality changes between this version and the next?

Comment (vmadura) [inline requirements.txt:9]: I am removing salt entirely from requirements.txt in my fork and testing bottle-saltmaster highstate by pulling salinity from my fork - Will update results in this PR.

Comment (vmadura) [inline requirements.txt:9]: Confirmed that the salt requirements was present only for the tests - bottle salinity was working even after removing this requirement. However, there are tests and for those we may need this dependency - in any case 2015.8.7 is the latest - so no further changes required.

**Pull request merged**